src/TheNextWeek/bvh.h

```diff
@@ -13,12 +13,20 @@
 
 #include "common/rtweekend.h"
 #include "hittable.h"
+#include <algorithm>
 
 
 class bvh_node : public hittable  {
     public:
-        bvh_node() {}
-        bvh_node(hittable **l, int n, double time0, double time1);
+        bvh_node() = delete;
+
+        bvh_node::bvh_node(hittable_list& list, double time0, double time1)
```

Comment:
This threw an error for me in clang
`bvh_node::bvh_node` -> `bvh_node`

It's complaining about the scope

Reply:
All gone.